app/models/invoice.rb

```diff
@@ -13,6 +13,8 @@ class Invoice < ApplicationRecord
   MAX_CHARGE_RETRIES = 3
   DECIMALS   = 2
   CHARGE_PRECISION   = 2
+  MIN_YEAR = 1980
```

Comment:
I am open for a discussion on the min and max values :)
Not sure in fact that we need to allow values before ~2010 (not sure when 3scale started to operate), but there are some tests with 1984, so I set it to 1980.

As for the max year, also not sure.

The main thing is for the range to be within 1000 (I don't think we want to increase that default)

Reply:
Oops, sorry, I thought it was in the error message I posted in the description, but it's not there.
1000 is the default for `:max_years_allowed'
https://apidock.com/rails/v5.1.7/ActionView/Helpers/select_year

This is used here for the years selector https://github.com/3scale/porta/blob/0e946cc325b60719fc3eff9820174b39da278765/app/views/finance/provider/invoices/index.html.erb#L28